### PR TITLE
UI-5 Installations Page components

### PR DESCRIPTION
**Pull request by julianjuko**
## Description 
Installations page components.

So far includes:

- Smart List
- Other components made functional

### Smart list screenshots in Storybook:
**Smart list as a list of custom JSX elements passed to it, with custom header**
image
**Smart list generated from data structure**
image
**No data [structure] supplied**
image

Pagination component will not be used for now, needs further refactoring to make into an independent control, completely decoupled from smart list and made dumber

### CODE REVIEW

File: src/PaginationPanel/PaginationPanel.tsx [hunk 1]
```diff
@@ -0,0 +1,161 @@
+import React, { useState, useEffect } from 'react'
```
Comment (julianjuko) [inline src/PaginationPanel/PaginationPanel.tsx]: This component won't be used yet, so can be refactored more later I guess. Has a few todos, see PR description

File: src/PaginationPanel/PaginationPanel.tsx [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+  edgesToShow = 2,
+  neighboursToShow = 1,
+  perPage = 5,
+  itemIds,
+  handleUpdate,
+  children
+}) => {
+  const [currentPage, setCurrentPage] = useState<number>(1)
+
+  const currentIds = itemIds.slice(
```
Comment (julianjuko) [inline src/PaginationPanel/PaginationPanel.tsx]: Will not need knowledge of current ids. Probably doesn't even need to receive `itemIds` as a prop anymore

File: src/PaginationPanel/PaginationPanel.tsx [hunk 3]
```diff
@@ -0,0 +1,161 @@
+import React, { useState, useEffect } from 'react'
+import { Button } from 'react-bootstrap'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faAngleLeft, faAngleDoubleLeft, faAngleRight, faAngleDoubleRight } from '@fortawesome/free-solid-svg-icons'
```
Comment (lmiller1990) [inline src/PaginationPanel/PaginationPanel.tsx]: If we want to use these icons in the pagination panel, we need update the readme and specify which peer dependencies the Cohesive UI relies on:

- react-bootstrap
- react v16 (with hooks)
- @fortawesome/react-fontawesome
- @fortawesome/free-solid-svg-icons

Comment (julianjuko) [reply src/PaginationPanel/PaginationPanel.tsx]: sure

**Pull request merged**